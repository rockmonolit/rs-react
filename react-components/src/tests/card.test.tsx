import * as React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';

import Card from '../components/card';

afterEach(cleanup);

it('this script should take a snapshot', () => {
  const { asFragment } = render(
    <Card author="test author" title="test title" description="test description" />
  );
  expect(asFragment()).toMatchSnapshot();
});
