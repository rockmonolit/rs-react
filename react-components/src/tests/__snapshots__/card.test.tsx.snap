// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="card"
  >
    <div
      class="cardContent"
    >
      <div
        class="cardImage"
      />
      <p
        class="titleText cardTitle"
      >
        test title
      </p>
      <p
        class="mainText cardAuthor"
      >
        test author
      </p>
      <hr />
      <p
        class="mainText cardText"
      >
        test description
      </p>
    </div>
  </div>
</DocumentFragment>
`;
